value/arguments/api.graphql

```diff
@@ -0,0 +1,43 @@
+# A `@value` directive defines a value that can be applied in various contexts
+# To establish a constant value within a type or query, we can utilize this directive.
+# If no arguments are provided (@value) then the value is null.
+
+type Customer {
+  name: String!
+  city: String!
+}
+
+# To establish a constant value for the state, utilize `@value` and provide the constant value.
+extend type Customer{
+  state:String @value(const:"Florida")
+}
+
+
+type Query {
+   emptyCustomer(id: ID): Customer @value # return null value
+   customer(id: ID): Customer # set the default value
```

Comment:
Be consistent with comment placement, others are before the field, not in the middle.

Update the comment similar to the others, the key point about this is that it is showing how to return a constant value for an object. It's not the default value.